tarantool/crud.py

```diff
@@ -0,0 +1,748 @@
+class Response(object):
+    def __init__(self, response):
+        """
+        TODO: write docs
+        """
+
+        if isinstance(response, dict):
+            for response_field_name in response.keys():
+                setattr(self, response_field_name, response[response_field_name])
+        elif isinstance(response, str):
+            self.str = response
+            self.err = response
+
+
+class Result(Response):
+    """
+    Contains result's fields from result variable of crud module operation.
+    """
+
+
+class Error(Response):
+    """
+    Contains error's fields from error variable of crud module operation.
+    """
+
+
+class Crud(object):
+    """
+    Contains crud module methods.
+    """
+
+    def __init__(self, con):
+        """
+        TODO: write docs
+        """
+
+        self.con = con
+        self._crud_module_init_complete = False
+
+    def _crud_module_init(self):
```

Comment:
https://github.com/tarantool/crud/blob/704bd6f1f1d6683906d800935450ef971ca4ccac/crud.lua#L176-L178

It seems that simply enabling crud router exposes crud handles to global. I've tried to experiment on it by myself and seems fine.

Moreover, doing explicit `crud = require('crud')` create global `crud` variable on storages (and in real clusters storages are mostly not routers) so this make this check more confusing (you can't call anything on storages).

Reply:
Done, now the require call for crud is not being made